test/conformance/kernel/urKernelGetGroupInfo.cpp

```diff
@@ -85,11 +87,25 @@ TEST_P(urKernelGetGroupInfoWgSizeTest, CompileWorkGroupSize) {
 }
 
 TEST_P(urKernelGetGroupInfoSingleTest, CompileWorkGroupSizeEmpty) {
-    // Returns 0 by default when there is no sepecific information
+    // Returns 0 by default when there is no specific information
     std::array<size_t, 3> read_dims{1, 1, 1};
     std::array<size_t, 3> zero{0, 0, 0};
     ASSERT_SUCCESS(urKernelGetGroupInfo(
         kernel, device, UR_KERNEL_GROUP_INFO_COMPILE_WORK_GROUP_SIZE,
         sizeof(read_dims), read_dims.data(), nullptr));
     ASSERT_EQ(read_dims, zero);
 }
+
+TEST_P(urKernelGetGroupInfoSingleTest, CompileMaxWorkGroupSizeEmpty) {
+    // Returns 0 by default when there is no specific information
+    std::array<size_t, 3> read_dims{1, 1, 1};
+    std::array<size_t, 3> zero{0, 0, 0};
+    auto result = urKernelGetGroupInfo(
+        kernel, device, UR_KERNEL_GROUP_INFO_COMPILE_MAX_WORK_GROUP_SIZE,
+        sizeof(read_dims), read_dims.data(), nullptr);
+    if (result == UR_RESULT_SUCCESS) {
+        ASSERT_EQ(read_dims, zero);
+    } else {
+        ASSERT_EQ(result, UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION);
```

Comment:
Please change this to a skip if the feature is unsupported. This will eliminate the need to include the test in the match files.

Reply:
This is the same pattern used elsewhere to handle the `UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION` is also success case.

Reply:
Oh, right, I just noticed a match file change and didn't properly read the test before commenting :)